discord/member.py

```diff
@@ -142,15 +142,15 @@ class Member(discord.abc.Messageable, _BaseUser):
     status : :class:`Status`
         The member's status. There is a chance that the status will be a :class:`str`
         if it is a value that is not recognised by the enumerator.
-    activity: Union[:class:`Game`, :class:`Streaming`, :class:`Activity`]
-        The activity that the user is currently doing. Could be None if no activity is being done.
+    activities: Tuple[Union[:class:`Game`, :class:`Streaming`, :class:`Activity`]]
+        The activities that the user is currently doing.
```

Comment:
“are currently doing”

Reply:
I mean that it should say "user are currently doing." rather than "… is currently doing.", since "activities" is plural.

Reply:
wait
i think i'm wrong here. i rescind my comment cause english is hard